configcatclient/configcatclient.py

```diff
@@ -139,10 +142,10 @@ def get_value_details(self, key, default_value, user=None):
         """
         settings, fetch_time = self.__get_settings()
         if settings is None:
-            message = 'Evaluating get_value_details(\'{}\') failed. Cache is empty. ' \
-                      'Returning default_value in your get_value_details call: [{}].'.format(key, str(default_value))
-            self.log.error(message)
-            details = EvaluationDetails.from_error(key, default_value, message)
+            message = 'Config JSON is not present. Returning the `%s` parameter that you specified in your application: \'%s\'.'
```

Comment:
Compare to the previous version here we lost info: Evaluating get_value_details(. is it acceptable?